src/functions.py

```diff
@@ -10,3 +10,12 @@
 
 # YOUR CODE HERE
 
+
+def is_even(num):
```

Comment:
The spec says is_even() should return a boolean value, so try to utilize the return value outside of the function to conditionally printout the appropriate output